defaults/03_community_operators.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: "operators.coreos.com/v1"
-kind: "OperatorSource"
+kind: "CatalogSource"
 metadata:
   name: "community-operators"
   namespace: "openshift-marketplace"
-  labels:
-    opsrc-provider: community
 spec:
-  type: appregistry
-  endpoint: "https://quay.io/cnr"
-  registryNamespace: "community-operators"
+  sourceType: grpc
+  image: registry.redhat.io/redhat/community-operator-index:v4.6
```

Comment:
I believe that we are changing this:

`quay.io/openshift-community-operators/catalog:latest`